README.md

```diff
@@ -1,59 +1,94 @@
-# Getting started with thread statistics on Mbed OS
+![](./resources/official_armmbed_example_badge.png)
+# Thread Statistics Mbed OS Example
 
-This guide reviews the steps required to get thread statistics on Mbed OS platform.
+This guide reviews the steps required to get thread statistics on an Mbed OS enabled platform.
 
-Please install [mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-## Import the example application
+## Application functionality
 
-From the command-line, import the example:
+The `main()` function outputs on the serial interface about a maximum of 8 threads information like `id`, `name`, `state`, `priority`, `stack_space`, `stack_size`.
```

Comment:
Suggestion: 
The `main()` function outputs on the serial interface thread information such as `id`, `name`, `state`, `priority`, `stack_space`, `stack_size`.